cumulus_library/study_parser.py

```diff
@@ -319,9 +319,11 @@ def _load_and_execute_builder(
         filename: str,
         cursor: databases.DatabaseCursor,
         schema: str,
-        verbose: bool,
+        verbose: bool = False,
         drop_table: bool = False,
         parser: databases.DatabaseParser = None,
+        generate_sql: bool = False,
```

Comment:
nit: is there a sensible way to reword this variable for clarity? Like even when this is False, SQL is still generated, right (I think)? It's actually whether that SQL is executed that is different between the two flows.

Reply:
`write_reference_sql` is more verbose but it's explicit.